CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1950,6 +1909,207 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float l
   return SiPixelTemplate::interpolate(id, cotalpha, cotbeta, locBz, locBx);
 }
 
+// *************************************************************************************************************************************
+//! Load template info for single angle point to invoke template reco for template generation
+//! \param      entry - (input) pointer to template entry
+//! \param      sizex - (input) pixel x-size
+//! \param      sizey - (input) pixel y-size
+//! \param      sizez - (input) pixel z-size
+// *************************************************************************************************************************************
+#ifdef SI_PIXEL_TEMPLATE_STANDALONE
+void SiPixelTemplate::sideload(SiPixelTemplateEntry* entry,
+                               int iDtype,
+                               float locBx,
+                               float locBz,
+                               float lorwdy,
+                               float lorwdx,
+                               float q50,
+                               float fbin[3],
+                               float xsize,
+                               float ysize,
+                               float zsize) {
+  // Set class variables to the input parameters
+  entry_sideloaded_ = entry;  // will bypass xtemp_[] and ytemp_[] and use those from this Entry.
+
+  enty1_ = entry;
+  enty0_ = entry;
+
+  entx00_ = entry;
+  entx02_ = entry;
+  entx20_ = entry;
+  entx22_ = entry;
+  entx21_ = entry;
+
+  Dtype_ = iDtype;
+  lorywidth_ = lorwdy;
+  lorxwidth_ = lorwdx;
+  xsize_ = xsize;
+  ysize_ = ysize;
+  zsize_ = zsize;
+  s50_ = q50;
+  qscale_ = 1.f;
+  for (int i = 0; i < 3; ++i) {
+    fbin_[i] = fbin[i];
+  }
+
+  // Set other class variables
+
+  // adcota_ = 0.f;
```

Comment:
These two can be removed, as no used anywhere

Reply:
removed